data/participants.json

```diff
@@ -1649,6 +1649,16 @@
 			"twitter"   : "",
 			"referral"  : ""
 
+		},
+	    	{
+			"name"      : "VIDUR KATARIA",
+			"imageur1"  : "assets/img/participants/DSC_8595.jpg",
```

Comment:
Also add image. Currently,there is no image with this name. And image name should be your username or firstname ...